gateway/gateway.go

```diff
@@ -917,34 +945,24 @@ func (gateway *HandleT) setWebPayload(r *http.Request, qp url.Values, reqType st
 }
 
 func (gateway *HandleT) pixelHandler(w http.ResponseWriter, r *http.Request, reqType string) {
-	if r.Method == http.MethodGet {
-		queryParams := r.URL.Query()
-		if writeKey, present := queryParams["writeKey"]; present && writeKey[0] != "" {
-			// make a new request
-			req, _ := http.NewRequest(http.MethodPost, "", nil)
 
-			// set basic auth header
-			req.SetBasicAuth(writeKey[0], "")
-			delete(queryParams, "writeKey")
+	queryParams := r.URL.Query()
+	writeKey := queryParams["writeKey"]
+	// make a new request
+	req, _ := http.NewRequest(http.MethodPost, "", nil)
```

Comment:
handle error here.

Reply:
error is ignored because we are ideally trying to send 200 with pixel value irrespective of any situation

Reply:
Server will crash if above statement fails with error. Handle the error and send 200.